src/mount/writedata.cc

```diff
@@ -596,49 +720,50 @@ void InodeChunkWriter::processDataChain(ChunkWriter& writer) {
 		}
 
 		// Let's sleep a bit shorter if we can't be woken up by the pipe to reduce the latency
```

Comment:
The comment looks outdated, unless the gWaveTimeout is set in other place.

Reply:
It is, comment removed.